javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,259 @@
+// --------------------------------------------------------------------------------   Movies exercise
+
+let allbadmov = [];
+let allbad2000 = [];
+let allbadtitle2000 = [];
+
+function getmovielist(data) {
+  // sessionStorage.clear();
+  let selectedvalue = document.getElementById("movlist").value;
+  // created an inputbox and a button for searching the gif file
+
+  if (selectedvalue == 0) {
+    alert("This can't be blank");
+    document.getElementById("movselected").innerHTML = `You should select some option to see results`;
+    document.getElementById("movresult").innerHTML = ``; 
+
+  } else if (selectedvalue == "allbad") {
+
+    console.log(selectedvalue);
+    data.forEach(element => {

```

Comment:
this can be done much quicker with filter. Try to use that instead

Reply:
Thanks, Done it 👍